### ISSUE TITLE
Support for time zone ID prefix

### ISSUE THREAD

**Issue by christopher-evans** [markuspoerschke/iCal#87]
Both [rfc2445](https://tools.ietf.org/html/rfc2445#section-4.2.19) and [rfc5545](https://tools.ietf.org/html/rfc5545#section-3.2.19) define an optional '/' prefix for the time zone.  Since the `Eluceo\iCal\Util\DateUtil` class is pulling the time zone out of the `DateTime` object there's no way to set the prefix.

**Comment by navarr** [comment]
Out of curiosity - is there a use case for adding this prefix?

**Comment by christopher-evans** [comment]
As far as I can see in the spec the prefix isn't actually _required_ when looking up against an external database/registry, but my interpretation was it should be used whenever the VTIMEZONE is defined externally, not in the ical message.

It's useful if you can set the time zone more generally (vs. the value given by `$dateTime->getTimezone()->getName()`).  If an app is using a time zone database e.g. from [here](https://github.com/libical/vzic) then the time zone would be set as something like `/citadel.org/database-version/Europe/London`.

**Comment by markuspoerschke** [comment]
Sure. Only the timezones supported by PHP are valid. Code like
```
$vEvent = new \Eluceo\iCal\Component\Event();
$vEvent->setDtStart(new \DateTime('2012-11-11 13:00:00', new \DateTimeZone('Mars/Colony')));
```
will result in an error.

I don’t see any use case at the moment. But if you want to create a merge request that will allow to override the used time zone, feel free to contribute! :)

**Comment by markuspoerschke** [comment]
Solved with #89
